index.html

```diff
@@ -55,5 +55,137 @@ <h1>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button">See project</button>
+            </div>
+          </div>
+  
+          <div class="card flex column">
+            <img src="./assets/card_bg.png" alt="img" />
+            <div class="card-details flex column">
+              <h2>Multi-Post-Stories</h2>
+              <h3>Gain &#43; Glory</h3>
+              <ul class="list flex">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button">See project</button>
+            </div>
+          </div>
+  
+          <div class="card flex column">
+            <img src="./assets/card_bg.png" alt="img" />
+            <div class="card-details flex column">
+              <h2>Multi-Post-Stories</h2>
+              <h3>Gain &#43; Glory</h3>
+              <ul class="list flex">
+                <li>Ruby on Rails</li>
+                <li>CSS</li>
+                <li>Javascript</li>
+                <li>HTML</li>
+              </ul>
+              <button type="button">See project</button>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="about-section">
+        <div class="about-me flex">
+          <h2>About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through some
+            of my work and experience! If you like what you see and have a project
+            you need coded, don’t hesitate to contact me.
+          </p>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="about-me-container">
+          <div class="about-me-item flex column">
+            <img src="./assets/Icon Langguage.svg" class="iconic" alt="img" />
+            <h3>Languages</h3>
+            <ul class="list skills flex">
+              <li>Javascript</li>
+              <li>Ruby</li>
+              <li>HTML</li>
+              <li>CSS</li>
+            </ul>
+          </div>
+  
+          <div class="about-me-item flex column">
+            <img src="./assets/framework.svg" class="iconic" alt="img" />
+            <h3>Frameworks</h3>
+            <ul class="list skills flex">
+              <li>React.js</li>
+              <li>Ruby on Rails</li>
+              <li>RSpec</li>
+              <li>Capybara</li>
+              <li>selenium</li>
+            </ul>
+          </div>
+  
+          <div class="about-me-item flex column">
+            <img src="./assets/skills.svg" class="iconic" alt="img" />
+            <h3>Skills</h3>
+            <ul class="list skills flex">
+              <li>Database Management</li>
+              <li>Version Control</li>
+              <li>CLI</li>
+              <li>Web Development</li>
+              <li>API Design</li>
+            </ul>
+          </div>
+        </div>
```

Comment:
Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
<div class="about-me-container">
  <ul class="list">
    <li>
      <div class="about-me-item flex column">
        <img src="./assets/Icon Langguage.svg" class="iconic" alt="img">
        <h3>Languages</h3>
        <ul class="list skills flex">
          <li>Javascript</li>
          <li>Ruby</li>
          <li>HTML</li>
          <li>CSS</li>
        </ul>
      </div>
    </li>

    <li>
      <div class="about-me-item flex column">
        <img src="./assets/framework.svg" class="iconic" alt="img">
        <h3>Frameworks</h3>
        <ul class="list skills flex">
          <li>React.js</li>
          <li>Ruby on Rails</li>
          <li>RSpec</li>
          <li>Capybara</li>
          <li>selenium</li>
        </ul>
      </div>
    </li>

    <li>
      <div class="about-me-item flex column">
        <img src="./assets/skills.svg" class="iconic" alt="img">
        <h3>Skills</h3>
        <ul class="list skills flex">
          <li>Database Management</li>
          <li>Version Control</li>
          <li>CLI</li>
          <li>Web Development</li>
          <li>API Design</li>
        </ul>
      </div>
    </li>
  </ul>
</div>

```

Consider updating your code following this structure and rules for semantics.